src/libs/API/index.ts

```diff
@@ -82,6 +91,7 @@ function write<TCommand extends WriteCommand>(command: TCommand, apiCommandParam
             // This should be removed once we are no longer using deprecatedAPI https://github.com/Expensify/Expensify/issues/215650
             shouldRetry: true,
             canCancel: true,
+            clientUpdateID: lastUpdateIDAppliedToClient,
```

Comment:
```suggestion
            lastUpdateID: lastUpdateIDAppliedToClient,
```
?

Reply:
I think I prefer clientUpdateID because we already use lastUpdateID in PHP and Auth. Sending it as lastUpdateID would just make things more confusing IMO